pkgs/by-name/so/sonar-scanner-cli/package.nix

```diff
@@ -0,0 +1,71 @@
+{
+  lib,
+  coreutils,
+  fetchFromGitHub,
+  jre,
+  makeWrapper,
+  maven,
+  nix-update-script,
+  unzip,
+  which,
+}:
+
+maven.buildMavenPackage rec {
+  pname = "sonar-scanner-cli";
+  version = "6.1.0.4477";
+
+  src = fetchFromGitHub {
+    owner = "SonarSource";
+    repo = "sonar-scanner-cli";
+    rev = "refs/tags/${version}";
+    hash = "sha256-RyuM1xzJD3PZV1EbAjmIZ7jIM9vPr7rORzGXa5IIdjk=";
+  };
+
+  mvnHash = "sha256-7kyY4S02NScQ9cRupxunB0JI7Z62c5nJMoHtcQkqfb4=";
+
+  mvnParameters = "-Dproject.build.outputTimestamp=1980-01-01T00:00:02Z";
+
+  nativeBuildInputs = [
+    coreutils
+    makeWrapper
+    unzip
+  ];
+
+  doCheck = false;
+
+  installPhase = ''
```

Comment:
Something in here is silently failing:

```
[WARNING] okio-3.6.0.jar, okio-jvm-3.6.0.jar define 1 overlapping resource: 
[WARNING]   - META-INF/okio.kotlin_module
[WARNING] maven-shade-plugin has detected that some files are
[WARNING] present in two or more JARs. When this happens, only one
[WARNING] single version of the file is copied to the uber jar.
[WARNING] Usually this is not harmful and you can skip these warnings,
[WARNING] otherwise try to manually exclude artifacts based on
[WARNING] mvn dependency:tree -Ddetail=true and the above output.
[WARNING] See https://maven.apache.org/plugins/maven-shade-plugin/
[INFO] Minimized 4531 -> 2998 (66%)
[INFO] Replacing original artifact with shaded artifact.
[INFO] Replacing /build/source/target/sonar-scanner-cli-6.1-SNAPSHOT.jar with /build/source/target/sonar-scanner-cli-6.1-SNAPSHOT-shaded.jar
[INFO] 
[INFO] --- assembly:3.7.1:single (cli) @ sonar-scanner-cli ---
[INFO] Reading assembly descriptor: src/main/assembly/noarch.xml
[INFO] Building zip: /build/source/target/sonar-scanner-6.1-SNAPSHOT.zip
[WARNING] Goal makeAggregateBom requires online mode for execution but Maven is currently offline, skipping
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  5.183 s
[INFO] Finished at: 2024-08-25T10:30:07Z
[INFO] ------------------------------------------------------------------------
Running phase: installPhase
error: builder for '/nix/store/hmi2lsf4z937dbbsmi86b4dhs9jknw36-sonar-scanner-cli-6.1.0.4477.drv' failed with exit code 1;
```